Code/Core/Time/Time.cpp

```diff
@@ -38,5 +38,13 @@ uint64_t Time::GetCurrentFileTime()
         #error Unknown platform
     #endif
 }
-
+//------------------------------------------------------------------------------
+uint64_t Time::FileTimeToSeconds(uint64_t _filetime)
+{
+    #if defined( __WINDOWS__ )
+        return _filetime / 10000000U;
+    #else
+        #error Unknown platform
```

Comment:
This will break compilation on OS X and Linux.

Reply:
Cache.cpp contains the necessary math to convert:
```C++
        #if defined( __WINDOWS__ )
            const uint64_t oneDay = ( 24 * 60 * 60 * (uint64_t)10000000 );
        #else
            const uint64_t oneDay = ( 24 * 60 * 60 * (uint64_t)1000000000 );
        #endif
```

Reply:
Fixed, sorry about that!